hazelcast-client-legacy/src/main/java/com/hazelcast/client/impl/ClusterAuthenticator.java

```diff
@@ -60,7 +60,7 @@ public void authenticate(ClientConnection connection) throws AuthenticationExcep
         //contains remoteAddress and principal
         SerializableList collectionWrapper;
         final ClientInvocation clientInvocation = new ClientInvocation(client, auth, connection);
-        final Future<SerializableList> future = clientInvocation.invoke();
+        final Future<SerializableList> future = clientInvocation.invokeUrgent();
```

Comment:
It might be better introducing a new interface like `UrgentClientRequest` and so `ClientInvocation` takes a request as urgent automatically if it implements `UrgentClientRequest`. 

I think, this way, when there is new client request that is needed to be sent urgent, it will be easier to integrate instead of changing the invocation call code to `invokeUrgent` from `invoke` in all places it is called.